src/SFA.DAS.Campaign.Web/Views/Industries/AgricultureEnvironmentAnimalCare.cshtml

```diff
@@ -32,4 +32,49 @@
     <dd>Land-based service engineering technician</dd>
     <dd>Sports turf operative</dd>
     <dd>Horticulture and Landscape Operative</dd>
-</dl>
\ No newline at end of file
+</dl>
+@section QuoteHero {
+    <section class="section section--slant section--apprentice-pale real-stories-quotes">
+        <div class="grid-row">
+            <div class="grid-column-two-thirds">
+                <div class="quote-feature">
+                    <h2 class="heading-xs quote-feature__caption">
+                        <span class="quote-feature__caption-text">Real stories</span>
+                        <span class="quote-feature__caption-icon"></span>
+                    </h2>
+                    <h3 class="heading-l">RSPCA</h3>
+
+                    <h4 class="heading-s">INCREASE YOUR TALENT POOL</h4>
```

Comment:
Sorry to be a pain, but can we ensure titles and text is in sentence case. Some screen readers treat uppercase as acronyms and read them letter by letter